src/appkit.cs

```diff
@@ -11867,6 +11868,16 @@ partial interface NSResponder : NSCoding {
 		[Mac (10,12)]
 		[Export ("newWindowForTab:")]
 		void GetNewWindowForTab ([NullAllowed] NSObject sender);
+
+		[Export ("presentError:")]
+		bool PresentError (NSError error);
+
+		[Export ("willPresentError:")]
+		NSError WillPresentError (NSError error);
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)] // You probably shouldn't override this method, because you have no way of reliably predicting whether this method vs. -presentError will be invoked for any particular error.
```

Comment:
Is this worth binding? I marked it advanced, and the docs suggest to pretty much ignore it and use WillPresentError.